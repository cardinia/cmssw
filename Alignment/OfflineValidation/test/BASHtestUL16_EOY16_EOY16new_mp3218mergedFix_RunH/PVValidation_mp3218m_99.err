27-Sep-2019 22:02:32 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root
27-Sep-2019 22:02:37 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root
Begin processing the 1st record. Run 284039, Event 774450, LumiSection 1 on stream 0 at 27-Sep-2019 22:03:05.115 CEST
Begin processing the 1001st record. Run 284039, Event 7803040, LumiSection 5 on stream 0 at 27-Sep-2019 22:16:06.074 CEST
Begin processing the 2001st record. Run 284039, Event 15088866, LumiSection 9 on stream 0 at 27-Sep-2019 22:20:22.194 CEST
Begin processing the 3001st record. Run 284039, Event 22156520, LumiSection 13 on stream 0 at 27-Sep-2019 22:24:37.025 CEST
Begin processing the 4001st record. Run 284039, Event 31470624, LumiSection 18 on stream 0 at 27-Sep-2019 22:28:46.953 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:28:51 CEST  Run: 284039 Event: 32111919
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 499
%MSG
Begin processing the 5001st record. Run 284039, Event 39456278, LumiSection 22 on stream 0 at 27-Sep-2019 22:32:54.210 CEST
Begin processing the 6001st record. Run 284039, Event 47021338, LumiSection 26 on stream 0 at 27-Sep-2019 22:37:01.888 CEST
Begin processing the 7001st record. Run 284044, Event 262010, LumiSection 1 on stream 0 at 27-Sep-2019 22:40:40.290 CEST
Begin processing the 8001st record. Run 284044, Event 10044090, LumiSection 7 on stream 0 at 27-Sep-2019 22:40:42.393 CEST
Begin processing the 9001st record. Run 284044, Event 17048620, LumiSection 12 on stream 0 at 27-Sep-2019 22:40:44.582 CEST
Begin processing the 10001st record. Run 284044, Event 24591264, LumiSection 17 on stream 0 at 27-Sep-2019 22:40:48.148 CEST
Begin processing the 11001st record. Run 284044, Event 31517160, LumiSection 22 on stream 0 at 27-Sep-2019 22:40:52.294 CEST
Begin processing the 12001st record. Run 284044, Event 40633312, LumiSection 28 on stream 0 at 27-Sep-2019 22:40:55.060 CEST
27-Sep-2019 22:40:55 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       1        1
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          284039/32111919                   
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 1                   1
System                  3                   3

dropped waiting message count 0
[8.526MB/8.526MB][100%][==================================================][8.526MB/s]  [8.526MB/8.526MB][100%][==================================================][8.526MB/s]  
