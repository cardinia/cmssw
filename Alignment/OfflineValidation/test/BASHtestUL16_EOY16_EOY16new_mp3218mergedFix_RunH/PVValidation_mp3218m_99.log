000 (3196555.099.000) 09/27 21:34:16 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196555.099.000) 09/27 21:44:23 Started transferring input files
	Transferring to host: <188.185.176.61:9618?addrs=188.185.176.61-9618+[2001-1459-301-5b--100-3c]-9618&noUDP&sock=17873_e530_73988>
...
040 (3196555.099.000) 09/27 21:44:23 Finished transferring input files
...
001 (3196555.099.000) 09/27 21:44:23 Job executing on host: <188.185.176.61:9618?addrs=188.185.176.61-9618+[2001-1459-301-5b--100-3c]-9618&noUDP&sock=17740_be30_3>
...
006 (3196555.099.000) 09/27 21:44:32 Image size of job updated: 1544
	2  -  MemoryUsage of job (MB)
	1472  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 21:49:33 Image size of job updated: 18252
	16  -  MemoryUsage of job (MB)
	15928  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 21:54:33 Image size of job updated: 31604
	29  -  MemoryUsage of job (MB)
	29256  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 21:59:33 Image size of job updated: 43272
	36  -  MemoryUsage of job (MB)
	36708  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 22:04:34 Image size of job updated: 484756
	446  -  MemoryUsage of job (MB)
	456092  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 22:09:35 Image size of job updated: 521372
	482  -  MemoryUsage of job (MB)
	492708  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 22:14:35 Image size of job updated: 600404
	558  -  MemoryUsage of job (MB)
	571024  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 22:19:35 Image size of job updated: 601820
	560  -  MemoryUsage of job (MB)
	572436  -  ResidentSetSize of job (KB)
...
006 (3196555.099.000) 09/27 22:24:36 Image size of job updated: 603172
	561  -  MemoryUsage of job (MB)
	573788  -  ResidentSetSize of job (KB)
...
040 (3196555.099.000) 09/27 22:41:30 Started transferring output files
...
040 (3196555.099.000) 09/27 22:41:30 Finished transferring output files
...
005 (3196555.099.000) 09/27 22:41:30 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:28:59, Sys 0 00:00:07  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:28:59, Sys 0 00:00:07  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4666  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	4666  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8773        1   3974538 
	   Memory (MB)          :      561     2000      2000 
...
