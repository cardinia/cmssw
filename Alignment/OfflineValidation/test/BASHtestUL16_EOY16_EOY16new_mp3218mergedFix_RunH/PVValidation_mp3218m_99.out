Job started at  Fri Sep 27 21:44:24 CEST 2019
batch dir: /pool/condor/dir_1936 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_284039_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.6M -rw-r--r--. 1 tkello zh 8.6M Sep 27 22:41 PVValidation_mp3218m_284039.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:44 PVValidation_mp3218m_284039_cfg.py -rw-r--r--. 1 tkello zh 3.4K Sep 27 22:41 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 27 21:54 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:44 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 22:02 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:44 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:44 var
Job ended at  Fri Sep 27 22:41:30 CEST 2019
